superset-frontend/plugins/plugin-chart-echarts/src/Timeseries/EchartsTimeseries.tsx

```diff
@@ -130,7 +130,9 @@ export default function EchartsTimeseries({
       }
       // Ensure that double-click events do not trigger single click event. So we put it in the timer.
       clickTimer.current = setTimeout(() => {
-        const { seriesName: name } = props;
+        const { seriesName: name, value } = props;
+        const xValue = value[0].getTime?.() || value[0];
+        console.log(xValue);
```

Comment:
we can remove it if this log is not needed

Reply:
Oops!